src/javascript/binary/pages.js

```diff
@@ -38,6 +38,35 @@ function get_login_page_url() {
     return 'https://' + page.settings.get('domains')['private'] + '/login' + params;
 }
 
+function is_valid_id(id){
+    return /^(MX|MF|VRTC|MLT|CR|FOG|VRTJ|JP)[0-9]+$/i.test(id);
+}
+
+function validate_cookies(){
+    if (getCookieItem('login')){
+        var accIds = $.cookie("loginid_list").split("+");
+        var loginid = $.cookie("loginid");
+        var logoutResponse = JSON.parse('{'+
+                                '"echo_req": {'+
+                                    '"logout": 1'+
+                                '},'+
+                                '"logout": 1,'+
+                                '"msg_type": "logout"'+
+                                '}');
```

Comment:
Please do not pass a fake response. This will not run the related back-end code. 
You need to just send a `logout` request via BinarySocket.